cli/src/factory_ops/boxing/init

```diff
@@ -95,6 +95,17 @@ fi
 # Prepare shutdown
 ensure_healthy_cluster
 
+echo "INFO: Resolving service user credentials" | tee -a ${LOG_FILE}
+# FIXME these values might be encrypted
+service_user_name=$(salt-call pillar.get system:service_user:name --output=newline_values_only)
+service_user_secret=$(salt-call pillar.get system:service_user:password --output=newline_values_only)
```

Comment:
Shouldn't we decrypt the password before using it?

Reply:
Yes, it's not done yet. I will address.

Reply:
done